enterprise_access/apps/subsidy_access_policy/models.py

```diff
@@ -784,7 +784,7 @@ def remaining_balance_per_user(self, lms_user_id=None):
         Returns the remaining redeemable credit for the user.
         """
         spent_amount = self.transactions_for_learner(lms_user_id)['aggregates'].get('total_quantity') or 0
-        return self.per_learner_spend_limit - spent_amount
+        return self.per_learner_spend_limit - int(spent_amount or 0)
```

Comment:
[clarification/curious] What situation leads to `spent_amount` being `None`? On line 786, if `self.transactions_for_learner(lms_user_id)['aggregates'].get('total_quantity')` were to return `None`, wouldn't `spent_amount` already be falling back to `0` due to the `or 0` on the previous line?

Reply:
That's what I thought too but I deployed the changes on stage today for learner credit subsidy box and got this [error](https://splunk.edx.org/en-US/app/search/search?sid=1694609260.2240668) from access service. 👀

Reply:
I see. I think the error is based on `self.per_learner_spend_limit` being `None`, not `spent_amount`. The `per_learner_spend_limit` model field can be `None` if the budget/policy type does not deal with a per-learner spend limit, where the math would be `None - 0`.